frontend/src/services/rule.service.ts

```diff
@@ -0,0 +1,12 @@
+import { httpService } from './http.service';
```

Comment:
Do we want rules and dictionaries in the same service?
How do we handle sources and segments (together or separate)?